sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,41 +1,53 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
+
+### New Features
+
+* Support for replaceable HTTP transport layer.
+* Add `version.hpp`.
 
 ### Breaking Changes
 
-* Move header `azure/storage/blobs/blob.hpp` to `azure/storage/blobs.hpp`
+* Move header `azure/storage/blobs/blob.hpp` to `azure/storage/blobs.hpp`.
+* Service API return types which are typically suffixed with `Result` are moved to the `Models` sub-namespaces and everything else from the protocol layer is made private by moving to the `Details` namespace.
+* Make XML serializer and deserializer private by moving them to the `Details` namespace.
+* Remove `BlockBlobClientOptions`, `AppendBlobClientOptions` and `PageBlobClientOptions`, use `BlobClientOptions` instead.
+* Rename `BlobSasBuilder::ToSasQueryParameters` to `BlobSasBuilder::GenerateSasToken`.
+
+### Other Changes and Improvements
+
+* Default uploading/downloading concurrency is changed from 1 to 5.
+* Remove support for specifying SAS version.
 
 ## 1.0.0-beta.4 (2020-10-16)
 
 ### Bug Fixes
```

Comment:
bug fixes go below new features and breaking changes sections.

Reply:
is that a big deal?

Reply:
Not a lot, but we do want to try to highlight the most important/impactful things here, and having similar section structure across the board helps that (also prevents subtle typos/issues where someone puts things in the wrong section accidentally).

See https://github.com/Azure/azure-sdk-for-cpp/pull/941#discussion_r521013885